src/cert.rs

```diff
@@ -15,8 +15,7 @@
 use crate::der::Tag;
 use crate::der::{self, CONSTRUCTED, CONTEXT_SPECIFIC};
 use crate::signed_data::SignedData;
-use crate::subject_name::GeneralName;
-use crate::x509::{remember_extension, set_extension_once, Extension};
+use crate::x509::{remember_extension, set_extension_once, DistributionPointName, Extension};
```

Comment:
Typo in your commit message: "that in `x509.rs`."

Reply:
Fixed.